src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,75 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class WorkWithFile {
+    private static final int OPERATION_NAME = 0;
+    private static final int OPERATION_AMOUNT = 1;
+    private static final String CSV_SEPARATOR = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] result = readFromFile(fromFileName);
+        writeToFile(toFileName, dataProcessing(result));
+    }
+
+    private String[] readFromFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        List<String> strings;
+
+        try {
+            strings = Files.readAllLines(Path.of(fromFileName));
+            if (strings.toString().equals("[]")) {
+                return new String[0];
+            }
```

Comment:
this check is not needed , empty string will be handled here as well strings.toArray(new String[0]);
let's remove it